src/shared/components/copyDownloadControls/CopyDownloadButtons.tsx

```diff
@@ -78,6 +80,27 @@ export class CopyDownloadButtons extends React.Component<
         );
     }
 
+    customButtons() {
+        // TECH: <If condition={this.props.showDownload}> was not working with returning multiple items in JSX.Element[], so moved the conditional here.
+        if (!this.props.showDownload) {
+            return null;
+        }
+
+        return getCustomButtonConfigs()
+            .filter(tool => tool.isAvailable?.() ?? true)
+            .map((tool, index: number) => {
+                return (
+                    <CustomButton
+                        key={tool.id}
+                        toolConfig={tool}
+                        baseTooltipProps={this.baseTooltipProps}
+                        downloadData={this.props.downloadData}
+                        overlayClassName={this.props.className}
+                    />
+                );
+            });
+    }
+
```

Comment:
We display the <CustomButton> components here following the Download button.  I couldn't get the "<If condition={this.props.showDownload}>" working in the HTML code so I moved the showDownload test to this function. I believe it's better that way anyways.